tests/python/unittest/test_operator.py

```diff
@@ -2303,6 +2303,200 @@ def check_output_n_grad(data_shape, idx_shape):
                 idx_shape += (np.random.randint(low=3, high=5), ) 
             check_output_n_grad(data_shape, idx_shape)
 
+def test_grid_generator():
+    # transform_type =  affine
+    test_case = [(20,21),(4,3),(6,12),(15,17)]
+    for target_shape in test_case:
+        affine_matrix =  mx.sym.Variable('affine')
+        grid = mx.sym.GridGenerator(data=affine_matrix,transform_type='affine', target_shape=target_shape)
+        exe = grid.simple_bind(ctx=mx.cpu(), affine=(1,6), grad_req='write')
```

Comment:
Try change "mx.cpu()" to "default_context()". By doing so we would be able to test the gpu code via `nosetests gpu/test_operator_gpu.py`